tests/controls/security_test.rb

```diff
@@ -0,0 +1,34 @@
+control "internal-security-1.0" do
+  impact 1.0
+  title "Tester for Openstack security rules."
+  desc "put description here"
+
+  username = user.username
+
+  describe sshd_config do
+    its('PermitRootLogin') { should eq 'no' }
+    its('PasswordAuthentication') { should eq 'no' }
+  end
+
+  # The brackets are needed because inspec is running the command, 
+  # so it shows up in addition to the ssd daemon
+  describe command('pgrep -f "[s]shd -D" | wc -w') do
```

Comment:
Is there a way to print a simpler message than the command itself in all the Inspec tests that are run ?

Like "Looking for running processes" instead of the command ?